packages/newspringchurchapp/src/ui/UserPrayer/index.js

```diff
@@ -13,20 +15,48 @@ const StyledView = styled(({ theme }) => ({
 }))(View);
 
 const UserPrayerList = () => (
-  <Query query={getUserPrayers} fetchPolicy="cache-and-network">
+  <Query query={getUserPrayers} fetchPolicy="network-only">
```

Comment:
Is this the only way for this query to work as intended? I don't like having to make a call everytime this view is presented.